src/sentry/api/bases/organization.py

```diff
@@ -201,6 +201,15 @@ class OrgAuthTokenPermission(OrganizationPermission):
     }
 
 
+class OrganizationMetricsPermission(OrganizationPermission):
```

Comment:
We needed different permissions since by default we don't have `org:read` on `POST`.

Reply:
We need to be careful though, since we don't want this endpoint to perform any write ops in the org, otherwise, it will effectively bypass permissions.